### ISSUE TITLE
what is found, missing, empty, and corrupted

### ISSUE THREAD

**Issue by alicera** [ultralytics/yolov5#9922]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

images and labels... 86733 found, 48364 missing, 0 empty, 15 corrupted: 100%

what is the define found, missing, empty, and corrupted ?
how to solove it ?

**Comment by alicera** [comment]
empty labels 
Does it mean convert yolo format error?

**Comment by FraCamp** [comment]
How to get labels? I have a custom dataset, with person COCO annotations

**Comment by alicera** [comment]
Do you have the share link?

**Comment by FraCamp** [comment]
Nope sorry, but I have all the images in a folder and I have the annotations (COCO version) in a json I believe I have to use some tool to get the labels or I have to commute the dataset into yolo format or something like that

**Comment by yingjunshi** [comment]
so what corrupted  means

**Comment by FraCamp** [comment]
I look it up, and looks like the script does not recognize the labels even tho they are there, this is an example of labels (named as: imagename.PNG -> imagename.txt)
![Screenshot 2023-03-09 alle 12 21 01](https://user-images.githubusercontent.com/74132335/224009058-71082ba4-dd2d-49a4-95a9-70f27a08cfa0.jpg)

**Comment by alicera** [comment]
I can't reproduce your case.
maybe, there are some error format on your data  .

**Comment by FraCamp** [comment]
Could you please tell me the data format? or at least could you direct me to a link or something like that where I can take a look at the data format for the folders (data/labels) and the labels format

**Comment by FraCamp** [comment]
Also the problem is when I use test.py with a custom dataset instead of COCO, in the datasets.py, utilized to configure the Pytorch dataloader, there is the all dataset that needs to be downloaded (datasets.py downloads COCO 2017 test train and val) and I just need to test the model "stock" with initial weights on my custom dataset (which is just a test set and has no train and val sets)

**Comment by alicera** [comment]
you can use coco128 
Just do it, it will download the coco128
```
python train.py
```

**Comment by glenn-jocher** [comment]
@alicera you can convert your COCO dataset annotations to the YOLO format using a conversion script provided in the YOLOv5 GitHub repository. You can find the script and an explanation of the YOLO label format in the official YOLOv5 documentation at https://docs.ultralytics.com/yolov5/.

For testing the model on a custom dataset, you can utilize the "coco128" example, as it's a small 128-image COCO-format dataset included with YOLOv5. You can run `python train.py` to quickly test the model on this dataset, even if you don't have train and validation sets.

Feel free to explore the official documentation for more detailed instructions on how to perform these tasks.
